it/connectors/PenaltiesConnectorISpec.scala

```diff
@@ -17,22 +17,32 @@
 package connectors
 
 
+import config.AppConfig
+import config.featureSwitches.{FeatureSwitching, UseAPI1811Model, UseAPI1812Model}
+import connectors.httpParsers.PenaltiesConnectorParser.GetPenaltyDetailsResponse
 import connectors.httpParsers.{InvalidJson, UnexpectedFailure}
-import models.User
+import models.{ETMPPayload, User}
 import org.scalatest.concurrent.ScalaFutures.convertScalaFuture
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import stubs.PenaltiesStub
 import stubs.PenaltiesStub._
 import testUtils.IntegrationSpecCommonBase
 import uk.gov.hmrc.http.HeaderCarrier
 
-class PenaltiesConnectorISpec extends IntegrationSpecCommonBase {
+class PenaltiesConnectorISpec extends IntegrationSpecCommonBase with FeatureSwitching {
```

Comment:
I think you can remove the FeatureSwitching trait here as you bring it in transitively from IntegrationSpecCommonBase
```suggestion
class PenaltiesConnectorISpec extends IntegrationSpecCommonBase {
```